webapp/views.py

```diff
@@ -356,7 +356,7 @@ def bulk_upsert_cve(*args, **kwargs):
 @marshal_with(MessageSchema, code=200)
 @marshal_with(MessageSchema, code=404)
 def delete_cve(cve_id):
-    cve = db_session.query(CVE).filter(CVE.id == cve_id).one_or_none()
+    cve = db_session.query(CVE).get(cve_id.upper())
```

Comment:
Note: Because testing this function requires authorization, this line was not tested before pushing this commit

Reply:
Ah! I think I've found a way for you to test this. Not manually in the browser, but using the tests.

In here:

https://github.com/mtruj013/ubuntu-com-security-api/blob/1316453ba09dcf08534678b88edf453a1447d74a/tests/test_routes.py#L186-L191

Try adding a test line or two with lowercase IDs, then run (with the Python venv enabled):

``` bash
dotenv flask run test-python
```
```